openstack/nova/templates/console-ingress.yaml

```diff
@@ -30,9 +30,12 @@ spec:
       {{- range $name, $config := .Values.consoles }}
         {{- if $config.enabled }}
       - path: /{{ $name }}/?(.*)
+        pathType: Prefix
```

Comment:
I think, that needs to be `ImplementationSpecific`, but quite honestly, I do not know why we even have the query parameters in the url-match

Reply:
You're misreading this. These are not query parameters but an optional `/`. We use this match in the annotations above to write a custom config into Nginx.

Reply:
IIRC, the console ingress was quite tricky in that we have multiple backends for it listening on different path-prefixes, but the backends themself don't understand these prefixes and need a rewrite to remove them. That's what the annotations do at least. But iirc, they stopped working with some nginx-ingress version and for that one, we might need the match here to get the actual path for the backends.

Reply:
Ah, you are right. And looking at the [examples](https://kubernetes.github.io/ingress-nginx/examples/rewrite/), `pathType: Prefix` is correct, but we might want to change the annotation to simply:
`nginx.ingress.kubernetes.io/rewrite-target: /$1`

Reply:
We have to be careful in changing the syntax here. @auhlig and I had some back-and-forth a while ago about an upgrade plan for the nginx-ingress and Nova's console ingress. I'm not sure where we are right now and if this is compatible. https://github.com/sapcc/helm-charts/commit/1c3ccd789bb5d13852e42278dbbc337a44ec5205 has some details

Reply:
Prefix works in qa-de-2. Fabian, do you want to discuss that other part with Arno and me or should we just merge this and move the discussion about the annotations to later?

Reply:
Sorry. Missed the 1st mention. But +1 for moving ahead to qa and fixing (if needed) the annotation there

Reply:
Let's move forward with this as it is. We can address the annotation in a different change (if needed)